deploy/example_catalog/cr-app-cdh632cm.json

```diff
@@ -120,20 +119,16 @@
                 "cmserver",
                 "primary",
                 "hive",
-                "worker",
-                "gateway",
-                "hbase",
-                "impala",
-                "apps"
```

Comment:
Are these intentional deletions? I still see role definitions for gateway, hbase, impala, and apps.

Reply:
@joel-bluedata no missed that, added all the other roles definitions now.